protocol/contracts/amo/Ramos.sol

```diff
@@ -316,12 +320,10 @@ contract Ramos is IRamos, TempleElevatedAccess, Pausable {
      * The remainder `quoteToken` are repaid to the recipient
```

Comment:
Should update the comment both here and in the Interface. No longer sending to 'recipient'